### ISSUE TITLE
[ShadyCSS] shadowRoot...innerHTML latency

### ISSUE THREAD

**Issue by dartess** [webcomponents/polyfills#25]
If in constructor used code like
```
this.shadowRoot.querySelector("section").innerHTML = ...
```
tags from string not styling.

For it work, need delay ~40ms, like this:
```
setTimeout(() => {
  this.shadowRoot.querySelector("section").innerHTML = ...
}, 40);
```

demo: https://codepen.io/anon/pen/ypjrvG?editors=1010

How can this be solved correctly?

**Comment by azakus** [comment]
This might be ShadyCSS or ShadyDOM, but I suspect this is fixed with the latest version of both, as they synchronously scope DOM now.

**Comment by dartess** [comment]
This was fixed in version 1.2.5. Thank you all!
